express/code/blocks/quotes/quotes.js

```diff
@@ -1,17 +1,256 @@
 // eslint-disable-next-line import/no-unresolved
-import { getLibs, addTempWrapperDeprecated } from '../../scripts/utils.js';
+import { getLibs, addTempWrapperDeprecated, getLottie, lazyLoadLottiePlayer, toClassName } from '../../scripts/utils.js';
+import loadCarousel from '../../scripts/utils/load-carousel.js';
+import {
+  fetchRatingsData,
+  determineActionUsed,
+  hasRated,
+  submitRating,
+  createRatingSlider,
+  sliderFunctionality,
+  RATINGS_CONFIG,
+  createHoverStarRating,
+  createRatingsContainer,
+} from '../../scripts/utils/ratings-utils.js';
 
 let createTag;
+let getConfig;
+let loadStyle;
+
+// Initialize utils
+let utils;
+try {
+  utils = await import(`${getLibs()}/utils/utils.js`);
+  ({ createTag, getConfig, loadStyle } = utils);
+} catch (error) {
+  window.lana?.log('Failed to load utils:', error, { tags: 'ax quotes' });
+  throw error;
+}
 
 function pickRandomFromArray(arr) {
   return arr[Math.floor(arr.length * Math.random())];
 }
 
+async function createQuotesRatings({
+  sheet,
+  isCarouselVariant = false,
+}) {
+  // Load placeholders module
+  const placeholders = await import(`${getLibs()}/features/placeholders.js`);
+
+  const data = await fetchRatingsData(sheet);
```

Comment:
Do we want to keep this all in since we broke the ticket apart?

Reply:
Looking at it more, I think it might be ok. But heads up @hadobe, as now we need to test this variation, re-test ratings block, all carousel functionality, past implementations of quotes blocks.